recipes-connectivity/openssh/README.md

```diff
@@ -0,0 +1,8 @@
+# OpenSSH with wolfSSL
+
+openssh_%.bbappend sets up the OpenSSH recipe to use wolfSSL instead of OpenSSL.
+wolfSSL should be configured with `--enable-openssh`. OpenSSH also needs to be
+patched, which is accomplished with the patch at
+files/wolfssl-openssh-8.5p1.patch. This patch comes from [the wolfSSL OSP (open
+source projects) repo](https://github.com/wolfSSL/osp). For other OpenSSH
+version patches, check out the OSP repo.
```

Comment:
I wonder if we can include better links here, perhaps leading an interested party directly to the patch directory (since we also have a full opnessh implementation in that repo) (https://github.com/wolfSSL/osp/tree/master/openssh-patches)

Also can we provide more of a "Quick Start Guide" or "how-to" so say a college student working on a semester project could fairly easily figure out what they would need to do to build this?

Something maybe that assumes they already have `bitbake core-image-minimal` succeeding from a guide like this one: https://tutorialadda.com/yocto/quick-start-your-first-yocto-project-build/ and then to pull in our solution on top of that they would need to do steps A, B, C, D, .... and then re-run `bitbake core-image-minimal` and then they're ready to test!

Reply:
> I wonder if we can include better links here, perhaps leading an interested party directly to the patch directory (since we also have a full opnessh implementation in that repo) ([https://github.com/wolfSSL/osp/tree/master/openssh-patches](https://github.com/wolfSSL/osp/tree/master/openssh-patches?rgh-link-date=2022-04-06T19%3A57%3A51Z))

Sure, I can update it.

> Also can we provide more of a "Quick Start Guide" or "how-to" so say a college student working on a semester project could fairly easily figure out what they would need to do to build this?

Isn't that what's covered in the meta-wolfssl README.md already?

Reply:
... And they would test it this way... Steps A, B...
A description of what a successful test  would look like.